compute/orchestration_test.go

```diff
@@ -185,6 +185,77 @@ func TestAccOrchestrationLifeCycle_update(t *testing.T) {
 	}
 }
 
+func TestAccOrchestrationLifeCycle_suspend(t *testing.T) {
+	helper.Test(t, helper.TestCase{})
+
+	orcClient, err := getOrchestrationsTestClients()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	instanceInput := &CreateInstanceInput{
+		Name:      _OrchestrationTestName,
```

Comment:
This is an interesting style. Are these internal identifiers? The standard go style is to make these lowercase (as I'd imagine is already known).

So instead of `_OrchestrationTestName` it would be `orchestrationTestName`, etc...

Seems like the underscore has been added here to sidestep this convention, which I don't necessarily think is good style.

Reply:
I can't find the exact reasoning behind this but it's centered around keeping these variables hidden. I'm having trouble finding the exact definition in docs but it's something to that effect.